proposals/lip-0061.md

```diff
@@ -106,25 +106,41 @@ Intuitively, the chain of trust property means that for any validator change a s
 
-The certificate generation specified in this LIP therefore generates an aggregate commit for any block at height `h` for which  `bftModule.existBFTParameters(h+1)` returns `True`. This means that at height `h+1` the [BFT module][lip-0058] stores new and possibly different BFT parameters that need to be authenticated by the block at height `h`. Note that the [DPoS module][lip-0057] and [PoA module][lip-0047] only set new BFT parameters for the height of the first block of a round, if the BFT parameters are different from the previous round. This implies that for DPoS or PoA an aggregate commit for the last block of a round is only required, if the next round uses different BFT parameters than the current round. As already mentioned, from the on-chain data, i.e., the aggregate commit and referenced block header, the corresponding certificates can be computed. Hence, the mechanism specified in this LIP guarantees that the generated certificates satisfy the chain of trust property.
+The certificate generation specified in this LIP therefore generates an aggregate commit for any block at height `h` for which  `existBFTParameters(h+1)` returns `True`, where [existBFTParameters][lip-0058#existbftparameters] is defined in LIP 0058. This means that at height `h+1` the [BFT store][lip-0058#bft-parameters] contains new and possibly different BFT parameters that need to be authenticated by the block at height `h`. As already mentioned, from the on-chain data, i.e., the aggregate commit and referenced block header, the corresponding certificates can be computed. Hence, the mechanism specified in this LIP guarantees that the generated certificates satisfy the chain of trust property.
 
 Note that if possible, also aggregate commits for blocks that are not at the end of the round are created and added to blocks so that a certificate is created as soon as a block is finalized and it is not required to wait for a change of BFT parameters.
 
 ## Specification
 
 ### Notation
 
-In this LIP, we write `bftModule.fct` for a call to the function `fct` defined in the [BFT module][lip-0058] and analogously `validatorsModule.fct` for a call to the function `fct` defined in the [Validators module][lip-0044].
+In this LIP, we will frequently use the following functions specified in LIP 0058 without reference to the function definition:
+
```

Comment:
I think we should add missing ` to all function names, but since you didn't now I wonder if it was a deliberate choice?

Reply:
Yep, makes sense. I fixed it everywhere.

Reply:
can be resolved